public/content/documentation/native/controls/link.md

```diff
@@ -0,0 +1,325 @@
+## General Notes
+
+How to test a link
+
+## Condensed
+
+### #a11y - Native App Accessibility Acceptance Criteria
+
+How to test a link
+
+1. Test keyboard only, then screen reader + keyboard actions
+
+   - Tab, arrow keys, or ctrl+tab: Focus visibly moves to the link
+   - Spacebar: Activates on iOS and Android
+   - Enter: Activates on Android
+
+2. Test mobile screenreader gestures
+
+   - Swipe: Focus moves to the element, expresses its name, role (state, if applicable)
+   - Rotor/talkback menu: Links can be navigated to and activated from the Rotor/TalkBack menu or by focus/double tap. Only one way is required. Known issue: Links do not currently appear in iOS Rotor using SwiftUI. 
+   - Doubletap: Activates the link
+
+3. Listen to screenreader output on all devices
```

Comment:
I feel like point 3. does not make sense for native app -- I'll add to agenda

Reply:
I've asked the group for feedback on this one. -- so feel free to wait until we get that!